### PR TITLE
Update LibraryThing Banner when logged out

### LINKED ISSUE #70: If you're not signed in to the VanBlack account, the banner should be a different colour
This is just to make sure that people can see what they're logged in as really quickly

### PR DESCRIPTION
**Pull request by braxtonhall**
toward #27 
resolves #70

### CODE REVIEW

File: extension/ts/extensions/banner.ts [hunk 1]
```diff
@@ -16,17 +23,21 @@ const setLogo = (id: string) =>
...
-	if (element) {
-		element.href = chrome.runtime.getURL("img/favicon.ico");
-	}
-};
+const setFavicon = () =>
+	Array.from(document.getElementsByTagName("link"))
+		.filter((element) => element.rel === "icon")
+		.map((element) => {
+			element.href = chrome.runtime.getURL("img/favicon.ico");
+			element.type = "image/x-icon";
```
Comment (adirar111) [inline extension/ts/extensions/banner.ts:31]: we expect this to only ever be a 1 element array after the filtering step right

File: extension/ts/extensions/banner.ts [hunk 2]
```diff
@@ -16,17 +23,21 @@ const setLogo = (id: string) =>
...
+		});
+
+const selectFilter = (): string =>
+	`saturate(${loggedIn() ? LOGGED_IN_SATURATION : LOGGED_OUT_SATURATION})`;
+
+const loggedIn = (): boolean => Cookies.get(LOGGED_IN_COOKIE_KEY) === LOGGED_IN_ID;
```
Comment (adirar111) [inline extension/ts/extensions/banner.ts:37]: 🍪

File: extension/ts/objects/cookies.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const cookies = (): Record<string, string> => {
+	const entries = document.cookie.split(";").map((entry) => entry.trim());
+	return Object.fromEntries(entries.map((entry) => entry.split("=")));
+};
```
Comment (adirar111) [inline extension/ts/objects/cookies.ts:4]: wizardry

**Review by adirar111 — APPROVED** [review]
noice!

File: extension/ts/extensions/banner.ts (hunk 1 above)
Comment (braxtonhall) [reply extension/ts/extensions/banner.ts:31]: No  It's possible to have multiple (which library thing sometimes does and it exposed a bug)

**Pull request merged**